requirements.txt

```diff
@@ -51,10 +51,14 @@ scipy==1.6.1
 scispacy==0.4.0
 sentence-transformers==0.4.1.2
 spacy==3.0.5
+spacy-transformers==1.0.1
+# spaCy 3.0.5 requires cupy-cuda102 < 9.0.0.
+cupy-cuda102==8.6.0
 typer==0.3.2
 srsly==2.4.0
 sphinx-bluebrain-theme==0.2.4
 torch==1.7.0
 tox==3.22.0
 tqdm==4.57.0
-transformers==4.3.2
+# spacy-transformers 1.0.1 requires transformers < 4.3.0.
+transformers==4.2.2
```

Comment:
I guess these new dependencies would also need to go into `setup.py` if we decide to incorporate this solution into our main branch. It's probably fine not to bother about it at the moment though.

Reply:
That was the subject of the discussion during the stand-up. Having them in `setup.py` is needed for `tox`.

Reply:
Done in https://github.com/BlueBrain/Search/commit/a3965e79f1a3f489b25fa5209498823ff53ab1e6.